packages/algorand-js/src/types.ts

```diff
@@ -70,3 +73,36 @@ export enum EncodingType {
   HEX,
   UTF8
 }
+
+export interface AccountsMap {
+  [addr: string]: SDKAccount
+}
+
+export interface State {
+  accounts: Map<string, SDKAccount>
+  globalApps: Map<number, SSCParams>
+}
+
+// describes interpreter's local context (state + txns)
+export interface Context {
+  state: State
+  tx: Txn // current txn
+  gtxs: Txn[] // all transactions
+}
+
+// represent account used in tests and by the context
+// NOTE: custom notations are used rather than SDK AccountState notations
+export interface SDKAccount {
```

Comment:
How about renaming `SDKAccount` to `StoreAccount`? This is to avoid a name clash with the `algosdk.Account`

Reply:
Good point. Thanks